lib/quicly.c

```diff
@@ -1097,7 +1097,9 @@ int quicly_get_stats(quicly_conn_t *conn, quicly_stats_t *stats)
     /* set or generate the non-pre-built stats fields here */
     stats->rtt = conn->egress.loss.rtt;
     stats->cc = conn->egress.cc;
-
+    if (stats->cc.cwnd_maximum < stats->cc.cwnd)
+        stats->cc.cwnd_maximum = stats->cc.cwnd;
```

Comment:
I'm a bit confused about what's being done here.

It seems that during slow start, `cwnd_maximum` as an internal value is kept UINT32_MAX, whereas the reported value will be the current CWND. Could you please elaborate why we cannot use the same definition for the two?

Reply:
The code basically records this value only on reductions, since that would happen less frequently. So the value of the cwnd when `get_stats()` is called might be larger than the maximum. 

I'll change this to simply record the value on cwnd increments, since that is more intuitive and less error prone, and it's not going to be noticeably more expensive.